tooling/css-variable-docgen-components/src/CharacteristicUsage.css

```diff
@@ -0,0 +1,14 @@
+.token {
+  font-family: var(--salt-typography-fontFamily-code);
+  height: var(--salt-size-base);
+  margin: var(--salt-size-unit);
```

Comment:
This style is breaking the code block. See Aria Announcer docs page

Reply:
thanks!